src/nodejs/server.ts

```diff
@@ -375,4 +399,89 @@ export class NstrumentaServer {
       logger.log('listening on *:' + port);
     });
   }
+
+  public async startLog(channels: string[]) {
+    if (this.activeChannelLogs.size > 0) {
```

Comment:
does this mean we can't start more than one log at a time?  seems like it might be useful to have e.g. a video log that is short in duration, but an always on log for something else like IMU data or step events, or similar